docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,11 @@
+Authentication
+===
+
+1. frontend: Sends user to the lock interface that's provided by auth0 when required.
+1. frontend: Lock interface returns a json web token.
+1. frontend: When making a request to the project service it sends the web token in the authorization header.
```

Comment:
So does the frontend tell the browser to store the token in local storage or as a cookie (so that it won't have to re-authenticate after a disconnection)?